src/mlpack/methods/kernel_svm/kernel_svm.hpp

```diff
@@ -72,15 +72,15 @@ class KernelSVM
    * @param data Input training features. Each column associate with one sample
    * @param labels Labels associated with the feature data.
    * @param regularization standard svm regularization parameter.
-   * @param fitIntercept true when not using linear kernel.
+   * @param fitKernel true when not using linear kernel.
```

Comment:
I don't think we should be using this. The user should specify only the kernel.
Also do we not need the `fitIntercept` boolean value.

Reply:
We do use `fitIntercept` with the `linearSVM` so IMO we should have it with the kernel_svm. Let me know what  you think.

Reply:
@himanshupathak21061998 can you make this change first? Also run the tests locally once